### DISCUSSION TITLE
Add a fall damage setting

### DISCUSSION THREAD

**Discussion by Tyguh** [BG-Software-LLC/SuperiorSkyblock2#1455] [Pending Feature Requests]
Hey, would it be possible for you to add a fall damage setting that is disabled by default? This will allow players to enable fall damage when they need to grind their acrobatics mcMMO and disable it when it's no longer needed.

**Comment by Tptuaazn** [comment]
https://wiki.bg-software.com/superiorskyblock/overview/island-flags#create-your-own-island-flag

**Reply by Tyguh** [reply]
I figured it would be a nice feature to come with the plugin instead of making an addon.

**Reply by Tptuaazn** [reply]
Fork this plugin and pull requests your added features.

**Comment by OmerBenGera** [comment]
It seems (for me at least) as a feature that should be in a separate addon rather than in the plugin.
You could easily hook it into the plugin using the Island flags API tho :)
